src/version.h

```diff
@@ -35,6 +35,7 @@ static const int NO_BLOOM_VERSION = 170004;
 
 //! Changes to CInv parsing, starting with this version:
 //! - MSG_WTX type defined, which contains two 32-byte hashes.
+//! - Undefined message types now return an error instead of being ignored.
```

Comment:
This documentation isn't quite accurate; nodes that _support_ this version will return an error for undefined message types, _regardless_ of negotiated protocol version. Once we've confirmed this is indeed the behaviour we want, I'll update the text.

Reply:
I don't think it is the behaviour we want. Normally we only change behaviour depending on the negotiated version (unless we're fixing a security bug), because that's safer in general and allows us to rely on the negotiated version being an accurate indicator when debugging, for example.

Reply:
I was adjusting the commits, so I removed this line while I was at it (adding an impl comment instead). If we decide to go the other way, I'll re-add it.